docs/spec_draft.md

```diff
@@ -339,6 +349,170 @@ only exists to establish data dependencies from `result` to `inputs`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.all_to_all
+
+### Semantics
+
+Scatters `split_count` blocks of data, by splitting `operand` along
+`split_dimension`, from all the execution instances to all the execution
+instances. Each execution instance concatenates the received blocks along the
+`concat_dimension`.
+
+The following diagram shows how the scattered blocks of `operand`, from all the
+execution instances, are concatenated to form `result` in each execution
+instance.
+
+<img align="center" src="spec_draft/all_to_all.svg" />
```

Comment:
The example operand is 2x4 but `operand` says 2x6, similarly for `split_count`.

Reply:
Thanks!